localplanning.services/src/lib/lpa-api.ts

```diff
@@ -10,6 +10,10 @@ export interface Service {
 interface LPA {
   name: string;
   slug: string;
+  theme: {
+    logo: string;
+    primary_colour: string;
```

Comment:
We should use camelCase throughout and handle casting at the query level.

```suggestion
    primaryColour: string;
```